mscore/toolbarEditor.ui

```diff
@@ -11,7 +11,7 @@
    </rect>
   </property>
   <property name="windowTitle">
-   <string>Dialog</string>
+   <string>Edit Toolbar</string>
```

Comment:
Maybe partially off-topic, but related: Is it possible that this one doesn't have a i18n key yet?

When going to "View → Toolbars → Customize Toolbars..." the window title always shows "Dialog" (no matter what language i set).

![grafik](https://user-images.githubusercontent.com/1634615/79706026-12400400-82b8-11ea-8ae3-9bccd42b4d64.png)

Reply:
That is what this PR is fixing